apps/kaizen/models.py

```
@@ -1,21 +1,17 @@
-import uuid
 from django.db import models
-from apps.accounts.models import User
 from common.models import BaseModel
 
 
 # Task model
 class Task(BaseModel):
     title = models.CharField(max_length=255)
-    description = models.TextField(blank=True, null=True)
-    priority = models.IntegerField()
+    details = models.TextField(blank=True, default="", null=True)
```

Ruff (0.8.2)

Avoid using `null=True` on string-based fields such as `TextField`

(DJ001)